test/controllers/project_skill_controller_test.exs

```diff
@@ -71,75 +60,49 @@ defmodule CodeCorps.ProjectSkillControllerTest do
 
       insert(:organization_membership, role: "admin", member: current_user, organization: organization)
```

Comment:
could the `role: "admin"` be eliminated by changing the `@tag :authenticated` to  `@tag authenticated: :admin` ? @JoshSmith, I'm not clear about this.